dojo/tools/crashtest_security/parser.py

```diff
@@ -48,13 +48,21 @@ def get_items(self, tree, test):
             if failure is None:
                 continue
 
-            title = node.get('name').upper()
+            title = node.get('name')
+            # Remove enumeration from title
+            title = re.sub(r' \([0-9]*\)', '', title)
+
+            # Attache CVEs
             if "CVE" in title:
                 cve = re.findall(r'CVE-\d{4}-\d{4,7}', title)[0]
```

Comment:
I would probably seize the chance here to modify the regex, and replace the `7` by a `10` to accommodate MITRE's number extension.

Reply:
Thanks for the good hint. I will update this.